tests/constants/data.constants.ts

```diff
@@ -0,0 +1,56 @@
+import { DataFile } from '../../src/app/core/models/data-file.model';
+import { Team } from '../../src/app/core/models/team.model';
+import { User, UserRole } from '../../src/app/core/models/user.model';
+
+// An example csv string that would be obtained from GitHub
+export const csvString = `
+role,name,team
+admin,admin,
+tutor,tutor1,CS2103T-W1-1
+tutor,tutor1,CS2103T-W1-2
+tutor,tutor2,CS2103T-W1-3
```

Comment:
Perhaps we should go with more realistic usernames, as real usernames may have quirks like special characters?